src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String randomColor = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(MAX_BALL_NUMBER + 1);
```

Comment:
This right now returns numbers from 0 to 100 inclusive, i think it is better to start from number 1 so:
`random.nextInt(MAXIMUM_NUMBER) + 1`